src/SourceGenerator/Xamarin.CommunityToolkit.SourceGenerator/XCTGenerator.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Xamarin.CommunityToolkit.SourceGenerator
+{
+	[Generator]
+	public class XCTGenerator : ISourceGenerator
+	{
+		const string code = @"
+namespace Xamarin.CommunityToolkit.Initializer
+{
+	sealed class XCTInitCaller
+	{
+		public void CallInit()
+		{
+			Xamarin.CommunityToolkit.Helpers.XCT.Init();
```

Comment:
what do you think if we add DrawingView.wpf after this PR and call its renderer here?

Reply:
Can you refresh my memory about it?

Reply:
There was an issue with Renderer initialization, and as workaround we need to call DrawingViewWpfRenderer.Init()

Reply:
Thanks. Yeah I remember that now. I can take a look right after we merge this one.